api/BatteryManager.json

```diff
@@ -0,0 +1,850 @@
+              ]
+            },
+            {
+              "version_added": "52",
+              "notes": [
+                "From this version onwards, the Battery Status API is only available in chrome/privileged code."
+              ]
+            }
+          ],
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "25"
+          },
+          "safari": {
+            "version_added": false
+          },
+          "webview_android": {
+            "version_added": "40"
+          },
+          "firefox_android": [
+            {
+              "version_added": "10",
+              "version_removed": "16",
+              "prefix": "moz"
+            },
+            {
+              "version_added": "16",
+              "version_removed": "52",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "dom.battery.enabled"
+                }
+              ],
+              "notes": [
+                "Disabled by default in Firefox 10.0, but can be enabled setting the preference <code>dom.battery.enabled</code> to <code>true</code>. Starting with Firefox 11.0, <code>mozBattery</code> is enabled by default. The Battery API is currently supported on Android, Windows, and Linux with UPower installed. Support for MacOS is available starting with Gecko 18.0 (Firefox 18.0 / Thunderbird 18.0 / SeaMonkey 2.15). Firefox also provide support for the deprecated <a href='https://developer.mozilla.org/docs/Web/API/Navigator/battery' title='The battery read-only property returns a BatteryManager provides information about the system's battery charge level; provides also some new events you can handle to monitor the battery status. This implements the Battery Status API; see that documentation for additional details, a guide to using the API, and sample code.'><code>navigator.battery</code></a>."
+              ]
+            },
+            {
+              "version_added": "52",
+              "notes": [
+                "From this version onwards, the Battery Status API is only available in chrome/privileged code."
+              ]
+            }
+          ],
+          "edge_mobile": {
+            "version_added": false
+          },
+          "opera_android": {
+            "version_added": "25"
+          },
+          "safari_ios": {
+            "version_added": false
```

Comment:
This one is a very special case: it has been added in an unknown version, and removed later in an unknown version (wasn't able to find which ones...)
So:
{
    "version_added": true,
    "version_removed": true
}